File: 3843/3843.c
int firstElementWithUniqueFrequenc(struct TreeNode* root) {
    if (!root) return 0;
    int l = firstElementWithUniqueFrequenc(root->left);
    int r = firstElementWithUniqueFrequenc(root->right);
    return 1 + (l > r ? l : r);
}

File: 3843/3843.cs
public class Solution {
    public int FirstElementWithUniqueFrequenc(TreeNode root) {
        if (root == null) return 0;
        return 1 + Math.Max(FirstElementWithUniqueFrequenc(root.left), FirstElementWithUniqueFrequenc(root.right));
    }
}